### PR TITLE
Make combobox and select dropdowns float above other UI

### PR DESCRIPTION
**Pull request by m-akinc**
# Pull Request

## 🤨 Rationale

Fixes: #606
Fixes: #1227

## 👩‍💻 Implementation

FAST fixed this issue with their combobox and select in their master branch, but we are still on FAST 1.0. In the PR for that change, they suggest that we essentially copy their implementation. See: https://github.com/microsoft/fast/pull/6452#issuecomment-1277956104. This PR does just that. We're using the package floating-ui to float and position the dropdown of the combobox and select. Because floating-ui now manages the position/size of the dropdown, the `position` and `maxHeight` properties are no longer functional. We had only exposed/documented the `position` property, and we had neglected to expose it from Angular. I've removed the properties from tests, Storybook, etc.

## 🧪 Testing

Some manual testing in Storybook. Existing unit tests pass.

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

File: change/@ni-nimble-blazor-1d363eda-276d-4735-9e97-20d631d96957.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (rajsite) [inline change/@ni-nimble-blazor-1d363eda-276d-4735-9e97-20d631d96957.json:1]: Was discussed at stand-up to explore the cost of using anchored-region for this use-case:

Anchored Region

Pros:
- Aligns with our existing pattern across controls (menu, menu-button, tooltip, is that all?)
- We aren't mixing popover implementations across components
- Has some more usage in the wild, i.e. the floating-ui library is part of FAST beta that hasn't released yet

Cons:
- Requires template forking
- Anchored region was proprietary to FAST and is deprecated, have to switch eventually.

If it's true that only a few of our controls use anchored region and two of three are already templated owned by us (tooltip and menu-button), then maybe we should be looking at the cost of migrating to floating ui for all of them

**Comment by atmgrifter00** [comment]
So, I've noticed in the [Combobox Storybook Doc page](https://60e89457a987cf003efc0a5b-jfqgyfrjkn.chromatic.com/?path=/docs/combobox--docs) that the dropdown is still clipped within its parent container. Were we not expecting this to be fixed with this change?

File: change/@ni-nimble-components-31e75753-f86e-4adf-8800-8c1ea12d516b.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (rajsite) [inline change/@ni-nimble-components-31e75753-f86e-4adf-8800-8c1ea12d516b.json:1]: Moving to threaded comment:
@atmgrifter00 
So, I've noticed in the [Combobox Storybook Doc page](https://60e89457a987cf003efc0a5b-jfqgyfrjkn.chromatic.com/?path=/docs/combobox--docs) that the dropdown is still clipped within its parent container. Were we not expecting this to be fixed with this change?

Comment (rajsite) [reply change/@ni-nimble-components-31e75753-f86e-4adf-8800-8c1ea12d516b.json:1]: looks like in chrome it pops over but firefox it's still stuck with the same storybook

Comment (m-akinc) [reply change/@ni-nimble-components-31e75753-f86e-4adf-8800-8c1ea12d516b.json:1]: I've run into enough problems and uncertainties that I'm planning on using anchored region for the select and combobox, instead. When that is ready, I'll close this PR and open a new one.

**Comment by m-akinc** [comment]
Closed but created another PR to address this issue: https://github.com/ni/nimble/pull/1244

**Pull request closed without merging**